### ISSUE TITLE
[Internal] Add a Dangerfile to this repo

### ISSUE THREAD

**Issue by AliSoftware** [CocoaPods/Xcodeproj#381]
Especially so that we ensure every PR has a CHANGELOG entry and correctly formatted (with full stop + 2 spaces + link to author & PR, etc)

\cc @orta

**Comment by orta** [comment]
You can grab the [ENV token](https://travis-ci.org/CocoaPods/CocoaPods/jobs/134565736) from the top of this build, then you ideally want the same [Dangerfile](https://github.com/CocoaPods/CocoaPods/blob/master/Dangerfile) as CocoaPods. It should take ~10m to set up, worth doing yourself you need to add it to the gemfile and run `bundle exec danger init` ( and giving me feedback )

**Comment by segiddins** [comment]
Maybe we want to make a CocoaPods danger plugin to centralize all our configuration across repos?

**Comment by orta** [comment]
IMO, I think that's probably for the best, I've just shipped 0.8 with the new plugins support, looks like https://github.com/dbgrandi/danger-proselint/blob/master/lib/danger_plugin.rb

We're still a bit off making `danger plugin create`

**Comment by AliSoftware** [comment]
Yeah about integration I was planning about trying to do that myself as I've never done it before and would be interested to learn how  to do it so that would be a good occasion :wink:
(But as my TODO list is already full I wanted to log the issue to be sure not to forget about it)

I pinged @orta just because I wonder if we already have a Danger file doing all that (not just checking for a CHANGELOG change but also checking the syntax and full-stop-two-spaces etc) and as I know we considered generalizing that solution (like a plugin dedicated to CHANGELOG checks?)
